source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,22 @@
+RoboRIO2 Imaging
+================
+
+The roboRIO2 uses a microSD card to hold all of it's software and robot code.
```

Comment:
There should probably be some type of intro to say it can be reimaged per the normal instructions, but reimaging the SD card is also an option